### PR TITLE
[content-service] allow to restrict the access to the signed URL

### PR DESCRIPTION
**Pull request by akosyakov**
#### What it does

gcloud storage requires exact content type match to allow access to bucket objects. This PR exposes the content type to allow internal service to have a control over it.

#### How to test

- Switch to Code.
- Test that code user data sync works against minio using this PR preview env.
- Test that code user data sync works against gcloud using preview env of https://github.com/gitpod-com/gitpod/pull/5163

### CODE REVIEW

**Comment by akosyakov** [comment]
Thanks @AlexTugarev for a hint!

File: components/content-service-api/typescript/src/blobs_pb.d.ts [hunk 1]
```diff
@@ -1,9 +1,3 @@
-/**
- * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
- * Licensed under the GNU Affero General Public License (AGPL).
- * See License-AGPL.txt in the project root for license information.
- */
-
```
Comment (corneliusludmann) [inline components/content-service-api/typescript/src/blobs_pb.d.ts:6]: I know, it's quite annoying that the generate script removes the license header. Please run
```
$ leeway run components:update-license-header
```
to add them again. Probably we should add this command to the generate script?

**Review by corneliusludmann — APPROVED** [review]
Besides the missing license headers, the code looks good so far.

**Comment by akosyakov** [comment]
I'm going to merge it to proceed with staging testing and work on other PRs. We can address any follow up with a follow up PR 😉

**Pull request merged**